collectors/feed/config.yml

```diff
@@ -0,0 +1,89 @@
+---
```

Comment:
How to solve the release-dependent changes potentially required in models then? See discussion https://github.com/RedHatProductSecurity/osidb/pull/367#discussion_r1383283922

Reply:
This is too complex to be in the scope of this task (I have no idea how to do that and if I did, I have no idea how to also make it work with local podman-compose). If you really want this, we can file a follow-up Task (I guess not a Story because I don't see end-user-visible value). Closing the conversation until then.